transifex/common/utils.py

```diff
@@ -6,14 +6,33 @@
 import pytz
 
 
-def generate_key(source_string, context=None):
+def generate_key(string, context=None, parsed=False, plurals=None):
```

Comment:
Minor: should we make `string` into a kwarg too?
So that you won't have to do `generate_key(None, plurals=...)`